CHANGELOG.md

```diff
@@ -158,7 +175,6 @@ These should not affect the functionality of the module.
 **Fixed bugs:**
 
 - rabbitmq\_user\_permissions timing out [\#689](https://github.com/voxpupuli/puppet-rabbitmq/issues/689)
-- Set default LC\_ALL =\> en\_US.UTF-8 \(\#671, \#689\) [\#694](https://github.com/voxpupuli/puppet-rabbitmq/pull/694) ([wyardley](https://github.com/wyardley))
```

Comment:
That this moves around could be a caching issue. Can you do a modulesync and use our fork to generate it?

Reply:
a modulesync from the master branch of modulesync_config should fix it

Reply:
Doing it, one sec

Reply:
Is this right now?